relayer/cli/src/commands/query/channel.rs

```diff
@@ -94,25 +110,14 @@ impl Runnable for QueryChannelEndCmd {
         };
         status_info!("Options", "{:?}", opts);
 
-        // run with proof:
-        // cargo run --bin relayer -- -c simple_config.toml query channel end ibc0 transfer ibconexfer
-        //
         // run without proof:
-        // cargo run --bin relayer -- -c simple_config.toml query channel end ibc0 transfer ibconexfer -p false
-        //
-        // Note: currently both fail in amino_unmarshal_binary_length_prefixed().
-        // To test this start a Gaia node and configure a channel using the go relayer.
+        // cargo run --bin relayer -- -c relayer/relay/tests/config/fixtures/simple_config.toml query channel end ibc-test firstport firstchannel --height 3 -p false
         let chain = TendermintChain::from_config(chain_config).unwrap();
-        let res = block_on(query_channel(
-            &chain,
-            opts.height,
-            opts.port_id.clone(),
-            opts.channel_id.clone(),
-            opts.proof,
-        ));
+        let res: Result<ChannelEnd, Error> = block_on(query(&chain, opts));
+
         match res {
-            Ok(cs) => status_info!("channel query result: ", "{:?}", cs.channel),
-            Err(e) => status_info!("channel query error: ", "{:?}", e),
+            Ok(cs) => status_info!("connection query result: ", "{:?}", cs),
```

Comment:
Typo? Let's keep "channel" instead of "connection" in these two lines.

Reply:
The curse of copy-paste